data/35e/wizards_of_the_coast/supplement/complete_arcane/car_spells.lst

```diff
@@ -9,27 +9,27 @@ Eldritch Blast								TYPE:Arcane		SCHOOL:Eldritch-Blast	SUBSCHOOL:Blast-Shape
 Devil's Sight								TYPE:Arcane		SCHOOL:Transmutation						DESCRIPTOR:Least-Invocation							COMPS:S	CASTTIME:1 standard action	RANGE:Personal		TARGETAREA:You																							DURATION:24 hours								SAVEINFO:None								SPELLRES:No																	SOURCEPAGE:p.133	DESC:You gain the visual acuity of a devil.  You can see normally in darkness and magical darkness up to 30 ft.
 Devour Magic								TYPE:Arcane		SCHOOL:Abjuration							DESCRIPTOR:Greater-Invocation							COMPS:S	CASTTIME:1 standard action	RANGE:Touch			TARGETAREA:Creature touched																					DURATION:Instantaneous							SAVEINFO:None								SPELLRES:No																	SOURCEPAGE:p.133	DESC:You deliver greater dispel magic by touch and gain 5 temporary hit points for each spell level dispelled. These temporary hit points do not stack with other temporary hit points.  If a new spell is devoured you may replace the old temporary hit points with the new ones, thus resetting the clock.
-Earthen Grasp (Invocation)	OUTPUTNAME:Earthen Grasp	TYPE:Arcane		SCHOOL:Transmutation						DESCRIPTOR:Least-Invocation|Earth						COMPS:S	CASTTIME:1 standard action	RANGE:Close			TARGETAREA:Animated earthen arm																				DURATION:(CASTERLEVEL*2) rounds					SAVEINFO:None								SPELLRES:Yes																SOURCEPAGE:p.133	DESC:You can cause an arm made of dense, compacted earth or soil (but not stone) to grapple your foes.  Treat as Medium creature with BAB +(CASTERLEVEL) and STR (floor(CASTERLEVEL/3)*2+14) the arm cannot move but can make one grapple attempt per round at any creature in its or adjacent square. Each round it successfully pins a target it deals 1d6+str lethal damage.  It has AC 15, hardness 4, (CASTERLEVEL*3) HP.
+Earthen Grasp (Invocation)	OUTPUTNAME:Earthen Grasp	TYPE:Arcane		SCHOOL:Transmutation						DESCRIPTOR:Least-Invocation|Earth						COMPS:S	CASTTIME:1 standard action	RANGE:Close			TARGETAREA:Animated earthen arm																				DURATION:(CASTERLEVEL*2) rounds					SAVEINFO:None								SPELLRES:Yes																SOURCEPAGE:p.133	DESC:You can cause an arm made of dense, compacted earth or soil (but not stone) to grapple your foes.  Treat as Medium creature with BAB +(CASTERLEVEL) and STR (floor(CASTERLEVEL/3)*2+14) the arm cannot move but can make one grapple attempt per round at any creature in it’s or adjacent square. Each round it successfully pins a target it deals 1d6+STR lethal damage.  It has AC 15, hardness 4, (CASTERLEVEL*3) HP.
```

Comment:
Mate, your correction is actually incorrect, as "it’s" is a contraction that means "it is", not a possessive which is normally implied with the "'s". The correct language is 'its' which is the possessive format by the publisher for their description.